meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh

```diff
@@ -260,6 +260,8 @@ board_config() {
         "NV4xPZ")
           DASHARO_REL_NAME="novacustom_nv4x_adl"
           DASHARO_REL_VER="1.7.2"
+          HEADS_REL_VER_DES="0.9.1"
```

Comment:
```suggestion
          HEADS_REL_VER_DES="0.9.0"
```

unless something changed?

Reply:
Nope, a leftover from testing the logic...